[BUG]: Heroku server crash

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
No STR at the moment, to view behavior:

1. Deploy code
2. Visit [https://fg-expense-bot.herokuapp.com/](https://fg-expense-bot.herokuapp.com/)
3. See error

**Expected behavior**
Server starts

**Screenshots**
```bash
2020-06-23T09:28:03.912670+00:00 app[web.1]: npm ERR! expenseng-bot@0.0.1 start: `node server.js`
2020-06-23T09:28:03.912847+00:00 app[web.1]: npm ERR! Exit status 1
2020-06-23T09:28:03.913039+00:00 app[web.1]: npm ERR! 
2020-06-23T09:28:03.913185+00:00 app[web.1]: npm ERR! Failed at the expenseng-bot@0.0.1 start script.
2020-06-23T09:28:03.913330+00:00 app[web.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-06-23T09:28:03.929871+00:00 app[web.1]: 
2020-06-23T09:28:03.930040+00:00 app[web.1]: npm ERR! A complete log of this run can be found in:
2020-06-23T09:28:03.930080+00:00 app[web.1]: npm ERR!     /app/.npm/_logs/2020-06-23T09_28_03_921Z-debug.log
2020-06-23T09:28:04.000001+00:00 heroku[web.1]: Process exited with status 1
2020-06-23T09:28:04.046135+00:00 heroku[web.1]: State changed from starting to crashed
2020-06-23T10:04:56.899006+00:00 heroku[web.1]: State changed from crashed to starting
2020-06-23T10:04:59.820775+00:00 heroku[web.1]: Starting process with command `npm start`
2020-06-23T10:05:03.116771+00:00 app[web.1]: 
2020-06-23T10:05:03.116785+00:00 app[web.1]: > expenseng-bot@0.0.1 start /app
2020-06-23T10:05:03.116786+00:00 app[web.1]: > node server.js
2020-06-23T10:05:03.116786+00:00 app[web.1]: 
2020-06-23T10:05:03.588011+00:00 app[web.1]: (●°u°●)​ 」 Bot starting...
2020-06-23T10:05:03.686182+00:00 app[web.1]: internal/modules/cjs/loader.js:1032
2020-06-23T10:05:03.686183+00:00 app[web.1]:   throw err;
2020-06-23T10:05:03.686184+00:00 app[web.1]:   ^
2020-06-23T10:05:03.686184+00:00 app[web.1]: 
2020-06-23T10:05:03.686185+00:00 app[web.1]: Error: Cannot find module 'Twit'
2020-06-23T10:05:03.686185+00:00 app[web.1]: Require stack:
2020-06-23T10:05:03.686186+00:00 app[web.1]: - /app/src/api/mentions.js
2020-06-23T10:05:03.686186+00:00 app[web.1]: - /app/server.js
2020-06-23T10:05:03.686189+00:00 app[web.1]:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
2020-06-23T10:05:03.686190+00:00 app[web.1]:     at Function.Module._load (internal/modules/cjs/loader.js:898:27)
2020-06-23T10:05:03.686191+00:00 app[web.1]:     at Module.require (internal/modules/cjs/loader.js:1089:19)
2020-06-23T10:05:03.686192+00:00 app[web.1]:     at require (internal/modules/cjs/helpers.js:73:18)
2020-06-23T10:05:03.686192+00:00 app[web.1]:     at Object.<anonymous> (/app/src/api/mentions.js:1:14)
2020-06-23T10:05:03.686193+00:00 app[web.1]:     at Module._compile (internal/modules/cjs/loader.js:1200:30)
2020-06-23T10:05:03.686193+00:00 app[web.1]:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
2020-06-23T10:05:03.686193+00:00 app[web.1]:     at Module.load (internal/modules/cjs/loader.js:1049:32)
2020-06-23T10:05:03.686199+00:00 app[web.1]:     at Function.Module._load (internal/modules/cjs/loader.js:937:14)
2020-06-23T10:05:03.686199+00:00 app[web.1]:     at Module.require (internal/modules/cjs/loader.js:1089:19)
2020-06-23T10:05:03.686200+00:00 app[web.1]:     at require (internal/modules/cjs/helpers.js:73:18)
2020-06-23T10:05:03.686200+00:00 app[web.1]:     at Object.<anonymous> (/app/server.js:7:21)
2020-06-23T10:05:03.686201+00:00 app[web.1]:     at Module._compile (internal/modules/cjs/loader.js:1200:30)
2020-06-23T10:05:03.686201+00:00 app[web.1]:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
2020-06-23T10:05:03.686202+00:00 app[web.1]:     at Module.load (internal/modules/cjs/loader.js:1049:32)
2020-06-23T10:05:03.686202+00:00 app[web.1]:     at Function.Module._load (internal/modules/cjs/loader.js:937:14) {
2020-06-23T10:05:03.686202+00:00 app[web.1]:   code: 'MODULE_NOT_FOUND',
2020-06-23T10:05:03.686203+00:00 app[web.1]:   requireStack: [ '/app/src/api/mentions.js', '/app/server.js' ]
2020-06-23T10:05:03.686204+00:00 app[web.1]: }
```

**Desktop (please complete the following information):**
- Heroku

**Smartphone (please complete the following information):**

**Additional context**
Add any other context about the problem here.

--- a/server.js
+++ b/server.js
@@ -3,7 +3,7 @@ const express = require('express');
 const search = require('./src/api/search');
 const authorize = require('./src/api/authorize');
 const callback = require('./src/api/callback');
-const like = require('./src/api/like')
+const like = require('./src/api/like');
 const getMentions = require('./src/api/mentions');
 const app = express();
 
@@ -13,19 +13,15 @@ app.get('/', home);
 app.get('/oauth/authorize', authorize);
 app.get('/oauth/callback', callback);
 app.get('/api/tweets/:query?', getTweets);
-app.get('/api/timeline/mentions', getMentions)
-app.post('/api/timeline/mentions',likeUsers)
+app.get('/api/timeline/mentions', getMentions);
+app.post('/api/timeline/mentions/like', likeUsers);
 
 function home(req, res) {
   res.sendFile(__dirname + '/src/views/index.html');
 }
 
 function likeUsers(req, res) {
-  //it likes users that tags it in hashtags(#) or mentions(@)
-  like(res)
-  // setInterval(() => {
-  //   like(res)
-  // }, 30000);
+  like(res);
 }
 
 function getTweets(req, res) {

--- a/src/api/like.js
+++ b/src/api/like.js
@@ -1,63 +1,53 @@
-const Twit = require('twit')
-const config = require('../config')
-const bot = new Twit(config.twitterKeys)
+const Twit = require('twit');
+const config = require('../config');
+const bot = new Twit(config.twitterKeys);
 
 //https://developer.twitter.com/en/docs/tweets/timelines/api-reference/get-statuses-mentions_timeline
-let like = (res) => {
-  get_mentions(res)
-}
+let like = res => {
+  get_mentions(res);
+};
 
-let get_mentions = (res) => {
-
-  bot.get('statuses/mentions_timeline', function (err, data) {
+let get_mentions = res => {
+  bot.get('statuses/mentions_timeline', function(err, data) {
     if (err) {
-      console.log('Error , can get your mentions', err)
+      console.log('Error , can get your mentions', err);
     } else {
       let mentions = [];
       if (!data || data.length == 0) {
-        res.status(200).json('No Mentions')
-        return
+        res.status(200).json('No Mentions');
+        return;
       }
       //res.status(200).json(data)
       data.forEach(e => {
-        mentions.push(e.id_str)
-      })
-      //res.status(200).json(mentions)    
+        mentions.push(e.id_str);
+      });
+      //res.status(200).json(mentions)
 
       //make sure the id of the users are not repeated
 
-      mentions = new Set(mentions)
+      mentions = new Set(mentions);
 
-      //Like 
-      favoriteTweet(mentions, res)
+      //Like
+      favoriteTweet(mentions, res);
     }
   });
-
-}
-
+};
 
 let favoriteTweet = async (data, res) => {
-  let errorMessage = []
+  let errorMessage = [];
   let result = await data.forEach(e => {
-    bot.post('favorites/create', { id: e }, function (err, response) {
-
+    bot.post('favorites/create', { id: e }, function(err, response) {
       if (err) {
-        errorMessage.push({ message: err.message, tweetId: e })
-        res.status(403).json(errorMessage)
+        errorMessage.push({ message: err.message, tweetId: e });
+        res.status(403).json(errorMessage);
         console.log('CANNOT BE FAVORITE... Error', err);
-      }
-      else {
-        res.status(200).json('Done', e)
+      } else {
+        res.status(200).json('Done', e);
         console.log('FAVORITED... Success!!!');
       }
     });
-    console.log(e)
+    console.log(e);
+  });
+};
 
-
-  })
-
-
-
-}
-
-module.exports = like
+module.exports = like;

--- a/src/api/mentions.js
+++ b/src/api/mentions.js
@@ -1,22 +1,20 @@
-const Twit = require('Twit');
+const Twit = require('twit');
 const config = require('../config');
-
-
 const bot = new Twit(config.twitterKeys);
 
-const getMentions =async (req, res, next) => {
+const getMentions = async (req, res, next) => {
   try {
     bot.get('statuses/mentions_timeline', (err, data, response) => {
-			if (err) {
-				throw err;
-			} else {
-				if (data.length < 1) {
-					res.status(404).json({ messgae: 'No mentions on timeline' });
-				} else {
-					res.status(200).json({ mentions: data });
-				}
-			}
-		});
+      if (err) {
+        throw err;
+      } else {
+        if (data.length < 1) {
+          res.status(404).json({ messgae: 'No mentions on timeline' });
+        } else {
+          res.status(200).json({ mentions: data });
+        }
+      }
+    });
   } catch (error) {
     res.status(500).json({
       message: 'Something went wrong',
@@ -24,7 +22,6 @@ const getMentions =async (req, res, next) => {
     });
     next(error);
   }
-}
-
+};
 
 module.exports = getMentions;

--- a/src/views/index.html
+++ b/src/views/index.html
@@ -23,15 +23,17 @@
           seperated strings.</li>
         <li><code>GET [domain_url]/oauth/authorize</code><br>a temporary endpoint for <code>@expenseng</code> to grant
           api access to thier Twitter account.</li>
-        <li><code>GET [domain_url]/api/timeline/mentions</code><br>bot can Like tweets where it is tagged(hashtaged(#) or
-          mentions(@)) </li>
+        <li><code>GET [domain_url]/api/timeline/mentions</code><br>timeline hashtags <code>#expenseng</code>and mentions
+          <code>@expenseng</code></li>
+        <li><code>GET [domain_url]/api/timeline/mentions/like</code><br>like Tweets that mentions or tags the bot</li>
       </ol>
 
       <h3>Click to view</h3>
       <ul>
-        <li><a href="api/tweets/%23expenseng">/api/tweets/%23expenseng</a></li>
-        <li><a href="oauth/authorize">/oauth/authorize</a></li>
-        <li><a href="/api/timeline/mentions">bot can Like tweets where it is tagged(hashtaged(#) or mentions(@)) </a></li>
+        <li><a href="api/tweets/%23expenseng">api/tweets/%23expenseng</a></li>
+        <li><a href="oauth/authorize">oauth/authorize</a></li>
+        <li><a href="api/timeline/mentions">api/timeline/mentions</a></li>
+        <li><a href="api/timeline/mentions/like">api/timeline/mentions/like</a></li>
       </ul>
     </div>
   </div>

